go.mod

```diff
@@ -8,8 +8,10 @@ require (
 	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
 	github.com/aead/chacha20poly1305 v0.0.0-20201124145622-1a5aba2a8b29 // indirect
 	github.com/fatih/color v1.13.0
+	github.com/fsnotify/fsnotify v1.5.4 // indirect
```

Comment:
Some new dependencies unrelated to the PR were added - can you check if they go away with `go mod tidy` ?